libopenage/util/misc.h

```diff
@@ -66,5 +73,95 @@ struct less {
 	}
 };
 
+
+// Size of uint64_s measured in bytes
+static constexpr size_t uint64_s = 8;
```

Comment:
In this case, using 8 is okay. Because i'm using it for conversions between uint8_t and uint64_t. I think that the variable name I used is misleading, maybe just replacing every ocurrence of uint64_s is better.